test/kpod.bats

```diff
@@ -21,15 +21,15 @@ function teardown() {
 	run ${KPOD_BINARY} ${KPOD_OPTIONS} pull debian:6.0.10
 	echo "$output"
 	[ "$status" -eq 0 ]
-	run ${KPOD_BINARY} $KPOD_OPTIONS rmi debian:6.0.10
+	run ${KPOD_BINARY} $KPOD_OPTIONS rmi docker.io/debian:6.0.10
 	[ "$status" -eq 0 ]
 }
 
 @test "kpod pull from docker without tag" {
 	run ${KPOD_BINARY} $KPOD_OPTIONS pull debian
 	echo "$output"
 	[ "$status" -eq 0 ]
-	run ${KPOD_BINARY} $KPOD_OPTIONS rmi debian
+	run ${KPOD_BINARY} $KPOD_OPTIONS rmi docker.io/debian:latest
```

Comment:
These should still work.

Reply:
yup

Reply:
I would prefer to add duplicate tests, where we do stuff with short name and long name.

Reply:
Yeah, those tests should really be for rmi though not put imho.